util/AuthorCollectionDefinition.ts

```diff
@@ -0,0 +1,10 @@
+/**
```

Comment:
Going to address where these live in [DEVED-74](https://sourcegraph.atlassian.net/browse/DEVED-74)